packages/host/app/components/operator-mode/card-preview-panel.gts

```diff
@@ -0,0 +1,212 @@
+      <div class='footer-buttons'>
+        <button
+          class='footer-button
+            {{if (eq this.previewFormat "isolated") "active"}}'
+          {{on 'click' (fn this.setPreviewFormat 'isolated')}}
+          data-test-preview-card-footer-button-isolated
+        >Isolated</button>
+        <button
+          class='footer-button
+            {{if (eq this.previewFormat "embedded") "active"}}'
+          {{on 'click' (fn this.setPreviewFormat 'embedded')}}
+          data-test-preview-card-footer-button-embedded
+        >
+          Embedded</button>
+        <button
+          class='footer-button {{if (eq this.previewFormat "edit") "active"}}'
+          {{on 'click' (fn this.setPreviewFormat 'edit')}}
+          data-test-preview-card-footer-button-edit
+        >Edit</button>
+      </div>
+    </div>
+
+    <style>
+      :global(:root) {
+        --code-mode-preview-footer-height: 70px;
+        --code-mode-preview-header-height: 70px;
+      }
+
+      .preview-header {
+        background: white;
+        height: var(--code-mode-preview-header-height);
+        border-top-left-radius: var(--boxel-border-radius);
+        border-top-right-radius: var(--boxel-border-radius);
+        padding: var(--boxel-sp-lg);
+        display: flex;
+      }
+
+      .header-icon > img {
+        height: 25px;
+        width: 25px;
+      }
+
+      .header-icon {
+        margin-right: var(--boxel-sp-xxs);
+      }
+
+      .preview-body {
+        height: calc(
+          100% - var(--code-mode-preview-footer-height) -
+            var(--code-mode-preview-header-height)
+        );
+      }
+
+      .header-actions {
+        margin-left: auto;
+      }
+
+      .preview-body :deep(.boxel-card-container) {
+        border-radius: 0;
+        box-shadow: none;
+        height: 100%;
+      }
+
+      /* hide the Header component that the card renderer adds in getBoxComponent, until we refactor it to not render it anymore. It shouldn't render it
+         because card creators might not want the header displayed when creating custom cards. */
+      .preview-body :deep(header) {
+        display: none;
+      }
+
+      .header-title {
+        font-weight: 600;
+        font-size: 1.2rem;
+      }
+
+      .preview-footer {
+        bottom: 0;
+        height: var(--code-mode-preview-footer-height);
+        background: #e8e8e8;
+        border-bottom-left-radius: var(--boxel-border-radius);
+        border-bottom-right-radius: var(--boxel-border-radius);
+      }
+
+      .footer-buttons {
+        margin: auto;
+        display: flex;
+        gap: var(--boxel-sp-sm);
+      }
+
+      .footer-button {
+        padding: var(--boxel-sp-xxxs) var(--boxel-sp-lg);
+        font-weight: 600;
+        background: transparent;
+        color: var(--boxel-dark);
+        border-radius: 6px;
+        border-color: #888888;
+        border-width: 1px;
+      }
+
+      .footer-button.active {
+        background: #262330;
```

Comment:
There two colors don't exist in our default color scheme. are they meant to be something else?